index.html

```diff
@@ -703,32 +696,31 @@
 
 ### Privacy Labour {#privacy-labour}
 
-<dfn data-lt="privacy labor|labour|labor">Privacy labour</dfn> is the practice of having a [=person=] carry out
+<dfn data-lt="privacy labor|labour|labor">Privacy labour</dfn> is the practice of having a [=person=] do
 the work of ensuring [=data processing=] of which they are the subject or recipient is
 [=appropriate=], instead of putting the responsibility on the [=actors=] who are doing the processing.
 Data systems that are based on asking [=people=] for their [=consent=] tend to increase
 [=privacy labour=].
 
-More generally, implementations of [=privacy=] are often dominated by self-governing approaches that
-offload [=labour=] to [=people=]. This is notably true of the regimes descended from the
-<dfn data-lt="FIPs">Fair Information Practices</dfn> ([=FIPs=]), a loose set of principles initially
-elaborated in the 1970s in support of individual [=autonomy=] in the face of growing concerns with databases. The
-[=FIPs=] generally assume that there is sufficiently little [=data processing=] taking place that any
-[=person=] will be able to carry out sufficient diligence to enable [=autonomy=] in their
-decision-making. Since they offload the [=privacy labour=]
-to people and assume perfect, unlimited [=autonomy=], the [=FIPs=] do not forbid specific
-types of [=data processing=] but only place them under different procedural requirements.
-This approach is no longer appropriate.
-
-One notable issue with procedural, self-governing approaches to privacy is that they tend to have the same
+More generally, implementations of [=privacy=] often offload [=labour=] to [=people=]. This is
+notably true of the regimes descended from the <dfn data-lt="FIPs">Fair Information Practices</dfn>
+([=FIPs=]), a loose set of principles initially elaborated in the 1970s in support of individual
+[=autonomy=] in the face of growing concerns with databases. The [=FIPs=] generally assume that
+there is sufficiently little [=data processing=] taking place that any [=person=] will be able to
+carry out sufficient diligence to be [=autonomous=] in their decision-making. Since they offload
+the [=privacy labour=] to people and assume perfect, unlimited [=autonomy=], the [=FIPs=] do not
+forbid specific types of [=data processing=] but only place them under different procedural
+requirements. This approach is no longer [=appropriate=].
```

Comment:
I'm still not sure it makes sense to complain about the FIPs here, but the introduction could be improved regardless:

```suggestion
<div class="example">
For example, the <dfn data-lt="FIPs">Fair Information Practices</dfn>
([=FIPs=]), a loose set of principles initially elaborated in the 1970s in support of individual
[=autonomy=] in the face of growing concerns with databases, generally assume that
there is sufficiently little [=data processing=] taking place that any [=person=] will be able to
carry out sufficient diligence to be [=autonomous=] in their decision-making. Since they offload
the [=privacy labour=] to people and assume perfect, unlimited [=autonomy=], the [=FIPs=] do not
forbid specific types of [=data processing=] but only place them under different procedural
requirements. This approach is no longer [=appropriate=].
</div>
```

Reply:
I don't think it is intended to be an example? It is a remark about how this consideration notably applies to the dominant paradigm. FIPs are the default assumption for most privacy lawyers and "pros", I think it's worth pointing out that they're not compatible.

Reply:
We'll resolve with https://github.com/w3ctag/privacy-principles/issues/347.